keyboards/bt66tech/bt66tech60/config.h

```diff
@@ -54,7 +54,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define RGB_DI_PIN B15
 #define RGBLED_NUM 9
 #define WS2812_SPI SPID2
-#define WS2812_SPI_MOSI_PAL_MODE 5
+#define WS2812_SPI_SCK_PIN B13
```

Comment:
This is definitely not correct — `B13` is in `MATRIX_COL_PINS`. Apparently the F103 chip does not require assigning the SCK function to a physical pin (`platforms/chibios/STM32_F103_STM32DUINO/board/board.h` does not seem to enable any SCK pins). Also `SPI2_SCK` is not even mappable to any other pin than `B13` on F103.